packages/visualizations/src/components/Map/utils.ts

```diff
@@ -119,11 +101,9 @@ function mergeBboxFromFeaturesWithSameKey(features: Feature[]) {
             feature.geometry.coordinates.forEach((coordsPath) => {
```

Comment:
It seems properly introspected in my VS Code somehow, and I don't have that warning. Maybe you have a different version of TS, with all the refactoring going on? But it would be easy to explicitly type this.

![Screenshot 2022-09-20 at 11 56 52](https://user-images.githubusercontent.com/1935200/191228224-c555fdfc-b48d-4930-8fe2-2a06c3594b1c.png)